assets/app/scripts/controllers/overview.js

```diff
@@ -58,6 +71,99 @@ angular.module('openshiftConsole')
       console.log("podsByServiceByLabel", $scope.podsByServiceByLabel);      
     };
 
+    var podRelationships = function() {
+      $scope.servicePodsNoDeployment = {};
+      $scope.solitaryPods = {};
+
+      angular.forEach($scope.pods, function(pod, name){
+        var replicationcontrollers = [];
+        var services = [];
+        angular.forEach($scope.deployments, function(deployment, depName){
+          var ls = new LabelSelector(deployment.spec.selector);
```

Comment:
if we're only doing this to see if this pod is addressed by any replication controllers, we can change `replicationcontrollers` to `inDeployment` and skip the loop once that's true